source/service.go

```diff
@@ -485,7 +485,7 @@ func extractServiceExternalName(svc *v1.Service) endpoint.Targets {
 	return endpoint.Targets{svc.Spec.ExternalName}
 }
 
-func extractLoadBalancerTargets(svc *v1.Service) endpoint.Targets {
+func ExtractLoadBalancerTargets(svc *v1.Service) endpoint.Targets {
```

Comment:
Why did you rename this function?
(Mostly) all other functions in source/service.go are starting with lowercase "extract...","get...".

Reply:
I needed to reuse this function at https://github.com/kubernetes-sigs/external-dns/pull/1725/files#diff-4b33f78c5676f61487528e03028e4567R337

Reply:
I can move this to a separate utils file as well and then refer that method in both places